plugins/score-card/src/api/ScoringDataJsonClient.ts

```diff
@@ -87,14 +87,26 @@ export class ScoringDataJsonClient implements ScoringDataApi {
         }
       },
     );
-    const entities = await this.catalogApi.getEntities({
-      filter: { kind: ['System'] },
+    if (!result) return undefined;
+
+    const entity_names: string[] = result.reduce((acc, a) => {
+      if (a.entityRef?.name) {
+        acc.push(a.entityRef.name);
+      }
+      return acc;
+    }, [] as string[]);
+
+    const response = await this.catalogApi.getEntities({
```

Comment:
I haven't look at pagination, not sure what happens when you have a lot (>1000 ?) of entity scores.

Reply:
Good point. In our case (Oriflame) since we do the scoring only for `system` kind it would be less than 100...I will create a new issue for this, ok?

Reply:
created [#141](https://github.com/Oriflame/backstage-plugins/issues/141)